app/controllers/occupations_controller.rb

```diff
@@ -12,8 +12,8 @@ def update
 
   private
 
-  def occupation_accepted?(params)
-    params[:commit].downcase != 'refuse'
+  def occupation_accepted?(_params)
+    action_params.fetch(:actions, []).any? { |action| action != 'no' }
```

Comment:
for added fun and magic, a bit of array coercion:
```
Array(action_params[:actions]).any? { |action| action != 'no }
```

not saying its better, just that its fun :tada: 

(fetch is probably clearer)